src/core/thread/mle_types.hpp

```diff
@@ -236,7 +236,7 @@ enum LeaderStartMode : uint8_t
  * Backbone Router / DUA / MLR constants
  *
  */
-constexpr uint16_t kRegistrationDelayDefault         = 1200;              ///< In seconds.
+constexpr uint16_t kRegistrationDelayDefault         = 5;                 ///< In seconds.
```

Comment:
@EskoDijk , any thoughts on whether a default registration delay should be specified by Thread?

Reply:
@jwhui Agree with the proposed reduction - it was supposed to be in the order of 5 seconds for typical/small Thread Networks. It should be as close as possible to "immediate" without causing a congestion.
A default could be defined by the Thread spec indeed; and a BR could deviate from the default when it has an implementation/algorithm to guide the choice of Reregistration Delay. For example a BR that knows there are 250 registered DUA Devices in its network can algorithmically increase the Delay to e.g. 20 seconds to spread the traffic.

Reply:
Created https://threadgroup.atlassian.net/browse/SPEC-1100 for this.

Reply:
Hi, Jonathon, EskoDijk has approved the changes, and I resolve this conversation.